EventListener/CorsListener.php

```diff
@@ -217,6 +220,20 @@ protected function getPreflightResponse(Request $request, array $options): Respo
 
         $response->headers->set('Access-Control-Allow-Origin', $origin);
 
+        // check private network access
+        if ($request->headers->has('Access-Control-Request-Private-Network')
+            && strtolower($request->headers->get('Access-Control-Request-Private-Network')) === 'true'
```

Comment:
```suggestion
        if (strtolower($request->headers->get('Access-Control-Request-Private-Network')) === 'true'
```

and, I'm not sure non lowercase `true` is allowed